plugins/importexport/pubmed/PubMedExportPlugin.inc.php

```diff
@@ -160,12 +160,22 @@ function exportIssues($issueIds, $context, $user) {
 		$pubmedExportFilters = $filterDao->getObjectsByGroup('article=>pubmed-xml');
 		assert(count($pubmedExportFilters) == 1); // Assert only a single serialization filter
 		$exportFilter = array_shift($pubmedExportFilters);
-		$submissionsIterator = Services::get('submission')->getMany([
-			'contextId' => $context->getId(),
-			'issueIds' => $issueIds,
-		]);
+
+		$input = [];
+		$sectionDao = DAORegistry::getDAO('SectionDAO');
+		foreach ($issueIds as $issueId) {
+			$sections = $sectionDao->getByIssueId($issueId);
+			$submissionsInSections = [];
```

Comment:
I think there is no need for this to be such an array -- I think it is enough to have `$submissionsInSections = Services::get('submission')->getInSections($issueId, $context->getId());` for each issue, no?